### PR TITLE
Add portuguese language

### PR DESCRIPTION
**Pull request by ViPeR5000**
add portuguese from Portugal language.

### CODE REVIEW

File: front-end/assets/translations/pt-pt.csv [hunk 1]
```diff
@@ -0,0 +1,195 @@
+Key;Translation
+ok;Ok
+select;Selecionar
+enabled;activar
+disabled;desativar
+cancel;cancelar
+done;Feito
+loading;Carregar
+documentation;Documenta��o
```
Comment (zekth) [inline front-end/assets/translations/pt-pt.csv]: Seems like there is some encoding problem in your file

**Comment by zekth** [comment]
Thanks for the contribution, but there is some encoding issues.

File: front-end/assets/translations/pt-pt.csv (hunk 1 above)
Comment (ViPeR5000) [reply front-end/assets/translations/pt-pt.csv]: Ups my mistake

new version

File: front-end/assets/translations/pt-pt.csv [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+alerts,Alertas
+unit,Unidades
+configure,Configurar
+language:language,Lingua
+language:fr,FranÃ§ais
+language:en,English
+language:de,Deutsch
+language:es,EspaÃ±ol
+language:it,Italiano
+language:hi,à¤¹à¤¿à¤¨à¥à¤¦à¥€
```
Comment (zekth) [inline front-end/assets/translations/pt-pt.csv]: ew?!

**Comment by ranjib** [comment]
Thank you @ViPeR5000

**Pull request merged**